src/ssl.c

```diff
@@ -8134,53 +8135,44 @@ int wolfSSL_i2d_PUBKEY(const WOLFSSL_EVP_PKEY *key, unsigned char **der)
     return wolfSSL_EVP_PKEY_get_der(key, der);
 }
 
-
-/* Reads in a DER format key. If PKCS8 headers are found they are stripped off.
- *
- * type  type of key
- * out   newly created WOLFSSL_EVP_PKEY structure
- * in    pointer to input key DER
- * inSz  size of in buffer
- *
- * On success a non null pointer is returned and the pointer in is advanced the
- * same number of bytes read.
- */
-WOLFSSL_EVP_PKEY* wolfSSL_d2i_PrivateKey(int type, WOLFSSL_EVP_PKEY** out,
-        const unsigned char **in, long inSz)
+static WOLFSSL_EVP_PKEY* wolfSSL_d2i_PublicKey_int(int type,
```

Comment:
Used the _int suffix for internal. What's the preferred naming scheme/syntax for functions like this one?

Reply:
I see a few cases where we used the suffix **_internal**. We also have cases where the internal version of the function is static and has an underscore prefix.

The suffix **_int**, from what I saw in a quick grep, is doing some processing with an integer value.